taccsite_cms/templates/snippets/manual-pattern-library/o-grid.content.html

```diff
@@ -0,0 +1 @@
+../../../static/site_cms/css/src/_imports/objects/o-grid.html
```

Comment:
__Explanation__
This is a symlink. (Templates can/should not be imported from outside of `templates` directory.[^1])

[^1]: If I am wrong, or there is a better solution, please tell me.